src/controllers/dynakube/deploymentmetadata/config.go

```diff
@@ -0,0 +1,28 @@
+package deploymentmetadata
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/logger"
+)
+
+const (
+	orchestrationTech = "Operator"
+
+	operatorScriptVersionKey = "script_version"
+	orchestratorIDKey        = "orchestrator_id"
+	orchestrationTechKey     = "orchestration_tech"
+
+	ApplicationMonitoringDeploymentType = "application_monitoring"
+	ClassicFullStackDeploymentType      = "classic_fullstack"
+	CloudNativeDeploymentType           = "cloud_native_fullstack"
+	HostMonitoringDeploymentType        = "host_monitoring"
+	ActiveGateDeploymentType            = "active_gate"
```

Comment:
These consts should come from `oneagent` and `activegate` to respect "open/closed principle".